cores/esp8266/core_esp8266_postmortem.cpp

```diff
@@ -36,41 +36,47 @@
 
 extern "C" {
 
-// These will be pointers to PROGMEM const strings
-static const char* s_panic_file = 0;
-static int s_panic_line = 0;
-static const char* s_panic_func = 0;
-static const char* s_panic_what = 0;
-
-// Our wiring for abort() and C++ exceptions
-static bool s_abort_called = false;
-static const char* s_unhandled_exception = NULL;
-
-// Common way to notify about where the stack smashing happened
-// (but, **only** if caller uses our handler function)
-static uint32_t s_stacksmash_addr = 0;
-
-void abort() __attribute__((noreturn));
-static void uart_write_char_d(char c);
-static void uart0_write_char_d(char c);
-static void uart1_write_char_d(char c);
-static void print_stack(uint32_t start, uint32_t end);
-
 // using numbers different from "REASON_" in user_interface.h (=0..6)
 enum rst_reason_sw
 {
     REASON_USER_STACK_SMASH = 253,
     REASON_USER_SWEXCEPTION_RST = 254
 };
-static int s_user_reset_reason = REASON_DEFAULT_RST;
+
+static struct PostmortemInfo {
+    int user_reset_reason;    // REASON_DEFAULT_RST;
+
+    // These will be pointers to PROGMEM const strings
+    const char* panic_file;   // 0;
+    int panic_line;           // 0;
+    const char* panic_func;   // 0;
+    const char* panic_what;   // 0;
+
+    // Our wiring for abort() and C++ exceptions
+    bool abort_called;        // false;
+    const char* unhandled_exception; // NULL;
+
+    // Common way to notify about where the stack smashing happened
+    // (but, **only** if caller uses our handler function)
+    uint32_t stacksmash_addr; // 0;
+} s_pm = {REASON_DEFAULT_RST, 0, 0, 0, 0, false, NULL, 0};
```

Comment:
Default construct in struct? Less comments, value would be right there near the name.
(c++ is magic)

Reply:
done